compiler/analyzer.ml

```diff
@@ -210,8 +217,22 @@ let constrain_ew env ew typ =
   match e with
     | Sast.Id(ssid) -> update_type env ssid typ; env, Sast.Expr(e, typ)
     | Sast.Fdecl(f) -> update_type env f.fname typ; env, Sast.Expr(e, typ)
+    | Sast.Call(Sast.Expr(Sast.Id(ssid), Sast.Func(func_typ)), _) -> 
```

Comment:
@akalicki Feel free to do whatever you would like for readability. there's another part where all the lines are like 90 chars, we might want to change that too.